include/bio_ik/frame.h

```diff
@@ -48,7 +48,7 @@ namespace bio_ik
 typedef tf2::Quaternion Quaternion;
 typedef tf2::Vector3 Vector3;
 
-struct Frame
+struct alignas(32) Frame
```

Comment:
What's the point of this explicit alignment? A comment would be nice here.

Reply:
`Frame` needs to be 32-byte aligned for the x86 intrinsics being done. I'll add a comment.

Reply:
commented in [c7edb0d](https://github.com/PickNikRobotics/bio_ik/commit/c7edb0dc0e4c107831cbbc116cc3f9547bb98564)